tests/__all__.pyi

```diff
@@ -0,0 +1,7 @@
+__all__: list[str]  # Y035 "__all__" in a stub file should be identical to "__all__" at runtime
+__all__: list[str] = ["foo", "bar", "baz"]
```

Comment:
There's not really much point in annotating `__all__` like this, but I don't really see a good reason to disallow it either.